### PR TITLE
Delete linting.yaml

### PR DESCRIPTION
**Pull request by jukent**
Per conversation in https://github.com/ProjectPythia/pythia-datasets/pull/41

I searched for other examples of the linting action across the organization. I think this is the last one.

### CODE REVIEW

**Comment by jukent** [comment]
Should we keep the linting badge in the README?

**Comment by andersy005** [comment]
> Should we keep the linting badge in the README?

we should replace it with the badge provided here: https://results.pre-commit.ci/repo/github/363016401

**Review by andersy005 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ The location can be overwritten by the `PYTHIA_DATASETS_DIR` environment
 variable to the desired destination.
 
 [github-ci-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/CI?label=CI&logo=github&style=for-the-badge
-[github-lint-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/linting?label=linting&logo=github&style=for-the-badge
+[pre-commit-badge]: https://results.pre-commit.ci/badge/github/ProjectPythia/pythia-datasets/main.svg
 [github-ci-link]: https://github.com/ProjectPythia/pythia-datasets/actions?query=workflow%3ACI
 [github-lint-link]: https://github.com/ProjectPythia/pythia-datasets/actions?query=workflow%3Alinting
```
Comment (andersy005) [inline README.md:84]:
```suggestion
[pre-commit-link]: https://results.pre-commit.ci/latest/github/ProjectPythia/pythia-datasets/main
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-| CI          | [![GitHub Workflow Status][github-ci-badge]][github-ci-link] [![GitHub Workflow Status][github-lint-badge]][github-lint-link] [![Code Coverage Status][codecov-badge]][codecov-link] |
-| :---------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| **Docs**    |                                                                    [![Documentation Status][rtd-badge]][rtd-link]                                                                    |
-| **Package** |                                                         [![Conda][conda-badge]][conda-link] [![PyPI][pypi-badge]][pypi-link]                                                         |
-| **License** |                                                                        [![License][license-badge]][repo-link]                                                                        |
+| CI          | [![GitHub Workflow Status][github-ci-badge]][github-ci-link] [![GitHub Workflow Status][pre-commit-badge]][github-lint-link] [![Code Coverage Status][codecov-badge]][codecov-link] |
```
Comment (andersy005) [inline README.md:1]:
```suggestion
| CI          | [![GitHub Workflow Status][github-ci-badge]][github-ci-link] [![GitHub Workflow Status][pre-commit-badge]][pre-commit-link] [![Code Coverage Status][codecov-badge]][codecov-link] |
```

File: README.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ The location can be overwritten by the `PYTHIA_DATASETS_DIR` environment
 variable to the desired destination.
 
 [github-ci-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/CI?label=CI&logo=github&style=for-the-badge
-[github-lint-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/linting?label=linting&logo=github&style=for-the-badge
+[pre-commit-badge]: https://results.pre-commit.ci/badge/github/ProjectPythia/pythia-datasets/main.svg
 [github-ci-link]: https://github.com/ProjectPythia/pythia-datasets/actions?query=workflow%3ACI
 [github-lint-link]: https://github.com/ProjectPythia/pythia-datasets/actions?query=workflow%3Alinting
 [codecov-badge]: https://img.shields.io/codecov/c/github/ProjectPythia/pythia-datasets.svg?logo=codecov&style=for-the-badge
```
Comment (andersy005) [inline README.md:85]: for consistency across all badges, remove the style:

```suggestion
[codecov-badge]: https://img.shields.io/codecov/c/github/ProjectPythia/pythia-datasets.svg?logo=codecov
```

File: README.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ The location can be overwritten by the `PYTHIA_DATASETS_DIR` environment
 variable to the desired destination.
 
 [github-ci-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/CI?label=CI&logo=github&style=for-the-badge
```
Comment (andersy005) [inline README.md:81]:
```suggestion
[github-ci-badge]: https://img.shields.io/github/workflow/status/ProjectPythia/pythia-datasets/CI?label=CI&logo=github
```

**Review by andersy005 — COMMENTED** [review]
@jukent, I left a few suggestions...  since the `pre-commit.ci` badge doesn't custom styling, option, it's worth making sure to remove `&style=for-the-badge` across the other badges

**Pull request merged**